src/name_bazaar/ui/db.cljs

```diff
@@ -117,3 +118,14 @@
      :saved-searches {:offerings-search {}}
      :watched-names {:ens/records {} :order '()}
      :infinite-list {:expanded-items {}}}))
+
+(defn try-resolving-address [db addr]
```

Comment:
this shouldn't be in `db.cljs`, just temporarily here? Correct place would be `name-bazaar.ui.utils`